modules/crm_tools/crm_tools.module

```diff
@@ -61,24 +61,21 @@ function template_preprocess_crm_tools_hat(array &$variables) {
  */
 function crm_tools_form_user_form_alter(&$form, FormStateInterface $form_state, $form_id) {
   if (isset($form['account']['roles'])) {
-    $tools = \Drupal::config('crm_tools.tools')->get('active_tools');
-    if (!empty($tools['hats'])) {
-      $form['account']['roles']['#attached']['library'][] = 'crm_tools/admin-forms';
-      $options = [];
-      /* @var \Drupal\crm_tools\AdvancedRoleStorageInterface $role_storage */
-      $role_storage = \Drupal::entityTypeManager()->getStorage('user_role');
-      if ($roles = $role_storage->loadTree()) {
-        foreach ($roles as $role) {
-          if ($role->id() == RoleInterface::ANONYMOUS_ID) {
-            continue;
-          }
-          $options[$role->id()] = str_repeat('-', $role->get('depth')) . Html::escape($role->label());
+    $form['account']['roles']['#attached']['library'][] = 'crm_tools/admin-forms';
+    $options = [];
+    /* @var \Drupal\crm_tools\AdvancedRoleStorageInterface $role_storage */
+    $role_storage = \Drupal::entityTypeManager()->getStorage('user_role');
+    if ($roles = $role_storage->loadTree()) {
+      foreach ($roles as $role) {
```

Comment:
We can probably just do `foreach ($role_storage->loadTree() as $role)`. If there are no roles, clearing the options will be empty anyway.